pusher-api/api-reference.md

```diff
@@ -0,0 +1,392 @@
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Get all the webhook subscriptions](#get-all-the-webhook-subscriptions)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Update a webhook subscription](#update-a-webhook-subscription)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Delete a webhook subscription](#delete-a-webhook-subscription)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Supported events](#supported-events)
+* [Examples](#examples)
+  * [Create a subscription](#create-a-subscription)
+  * [Get subscriptions](#get-subscriptions)
+  * [Update a subscription](#update-a-subscription)
+  * [Delete a subscription](#delete-a-subscription)
+
+
+### Base URL
+https://pusher.izettle.com
+
+### OAuth Scope
+In order to create or update a subscription to an event, you will need to be authorized with the corresponding scope.
+
+E.g. you will require the `READ:PURCHASE` scope if you want to subscribe to the `PurchaseCreated` event. See the [list of scopes](#supported-events) corresponding to every event.
+
+See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information on how to get authorization for a particular scope.
+
+## Create a webhook subscription
+
+Creates a webhook subscription to a specific event. 
+
+Once the subscription for an event gets created successfully, Pusher service will publish data on the integrator's service whenever that event gets triggered.<br/>
+E.g. Let's say that you created a subscription for the ```ProductUpdated``` event. Once this is done, whenever a product gets updated in the product library and the `ProductUpdated` event gets triggered, you will receive event data i.e, payload for the updated product on the ```destination``` that you have exposed publicly.
+See a list of [payloads](user-guides/event-subscriptions.md#payloadAPITable) for all events.
+
+The service will push data for an event only once. However, there may be cases where it gets published more than once. The integrator will then have to take care to not save the data more than once.
+
+```
+POST /organizations/{organizationUuid}/subscriptions
+```
+
+See [Create a subscription example](#create-a-subscription).
+
+
+### Parameters
+
+<details><!-- start tag of the Parameters section-->
+<summary>Click to see all request parameters for creating a subscription.</summary>
+
+|Name |Type |In |Required/Optional |Description
+|---- |---- |---- |---- |----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can use following options to fill in this value: <br/><ul><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li><li> Use `self` as the value. This will retrieve your organizationUuid through the authentication information in the request.</li></ul> 
```

Comment:
Maybe switch order of the options in the list?
Much easier ton use self than call `https://oauth.izettle.com/users/me`.

And maybe 
`This will retrieve organizationUuid from the authentication token.`